ui/rewards/dashboard/RewardsDashboardActivitySection.tsx

```diff
@@ -0,0 +1,209 @@
+            amountDiff: activities.transactions?.amountDiff,
+          },
+          {
+            title: 'Weekly contracts verification',
+            description: (
+              <>
+                Grab your{ ' ' }
+                <Link external href={ `${ activityPassUrl }&utm_medium=verify-contracts-task` }>
+                  Activity pass
+                </Link>{ ' ' }
+                then{ ' ' }
+                <Link href={ route({ pathname: '/contract-verification' }) }>
+                  verify smart contracts
+                </Link>{ ' ' }
+                manually on Blockscout for different chains and earn extra Merits every week.
+              </>
+            ),
+            percentileHint: 'Measures your performance relative to other Merit program participants. Verify more contracts to increase your weekly performance rank.', /* eslint-disable-line max-len */
+            amountHint: 'The number of extra Merits you have earned this week from your contract verifications. Verify more contracts to earn additional Merits.', /* eslint-disable-line max-len */
+            percentile: activities.contracts?.percentile,
+            percentileDiff: activities.contracts?.percentileDiff,
+            amount: activities.contracts?.amount,
+            amountDiff: activities.contracts?.amountDiff,
+          },
+          {
+            title: 'Weekly Blockscout usage',
+            description: (
+              <>
+                Grab your{ ' ' }
+                <Link external href={ `${ activityPassUrl }&utm_medium=usage-task` }>
+                  Activity pass
+                </Link>{ ' ' }
+                then just use Blockscout explorers in your every day routine.
+              </>
+            ),
+            percentileHint: 'Measures your performance relative to other Merit program participants. Use Blockscout explorers and add them to a MetaMask wallet to increase your weekly performance rank.', /* eslint-disable-line max-len */
+            amountHint: 'The number of extra Merits you have earned this week from your Blockscout usage. Use Blockscout explorers and add them to a MetaMask wallet to earn additional Merits.', /* eslint-disable-line max-len */
+            percentile: activities.usage?.percentile,
+            percentileDiff: activities.usage?.percentileDiff,
+            amount: activities.usage?.amount,
+            amountDiff: activities.usage?.amountDiff,
+          },
+        ].map((item, index) => (
+          <RewardsDashboardCard
+            key={ index }
+            label={ period }
+            title={ item.title }
+            description={ (
+              <Flex flexDir="column" gap={ 3 }>
+                <Box wordBreak="break-word">{ item.description }</Box>
+                <Flex alignItems="center" gap={ 3 }>
+                  <Button
+                    flex={{ base: 1, md: 'none' }}
+                    loading={ instancesQuery.isLoading }
```

Comment:
```suggestion
                    loadingSkeleton={ instancesQuery.isLoading }
```

It might be more consistent with other UI elements if we cover this button with a skeleton while loading occurs.